marshmallow/utils.py

```diff
@@ -334,16 +334,31 @@ def callable_or_raise(obj):
     return obj
 
 
-def get_func_args(func):
```

Comment:
I think I agree with maximkulkin and I programmed it and will send a pull request. I think putting the two functions into compat will be the cleanest solution, sind compat is about branching and compat may be imported by utils but not the other way around.